### ISSUE TITLE
Inference example code for single video input

### ISSUE THREAD

**Issue by wzmsltw** [facebookresearch/video-nonlocal-net#4]
Hi Xiaolong
Thank you for this outstanding work about Non Local network!
Since I want to test Kinetics-pretrained non-local network on other dataset (such as UCF-101) to extract feature, will you provide a inference example which read in a single video clip and output softmax score?

Thanks so much!

**Comment by Zumbalamambo** [comment]
+1

**Comment by gemfield** [comment]
You may find a solution here: https://zhuanlan.zhihu.com/p/36045849

**Comment by gemfield** [comment]
@wzmsltw You may find a solution here: https://zhuanlan.zhihu.com/p/36045849

**Comment by hengshanji** [comment]
@gemfield  Hi 想问下加载训练好的model 有更简便的方法吗？ 作者加载模型的代码太复杂了，而且保持的不是pb格式的模型

**Comment by jayleicn** [comment]
Is there anyone succeed in using this code for feature extraction?

**Comment by Zumbalamambo** [comment]
@jayleicn this is just a theoretical concept so practically not possible
